docs/source/framework.rst

```diff
@@ -12,48 +12,53 @@ Framework
 Configuration
```

Comment:
This document needs a complete re-write given the new configurator. I will do this.

Reply:
ok, thanks!